src/Magnum/Audio/Renderer.h

```diff
@@ -58,6 +61,37 @@ class Renderer {
             OutOfMemory = AL_OUT_OF_MEMORY  /**< Unable to allocate memory */
         };
 
+        enum class HrtfStatus: Short {
+            Disabled = ALC_HRTF_DISABLED_SOFT,    /**< HRTF is disabled */
+            Enabled = ALC_HRTF_ENABLED_SOFT,      /**< HRTF is enabled */
+            /**
+             * @brief HRTF is disabled because it is not allowed on the device
+             *
+             * This may be caused by invalid resource permissions, or an other
+             * user configuration that disallows HRTF.
+             */
+            Denied = ALC_HRTF_DENIED_SOFT,
+            /**
+             * @brief HRTF is enabled because it must be used on the device
+             *
+             * This may be caused by a device that can only use HRTF, or other
+             * user configuration that forces HRTF to be used.
+             */
+            Required = ALC_HRTF_REQUIRED_SOFT,
+            /**
+             * @brief HRTF is enabled automatically because the device reported
+             * headphones.
+             */
+            Detected = ALC_HRTF_HEADPHONES_DETECTED_SOFT,
+            /**
+             * @brief The device does not support HRTF with the current format
```

Comment:
Please don't use `@brief` in enum value docblocks, it does weird things to the generated documentation ... so just the sentence with a period at the end.

Sorry again for nitpicking :)